docs/user/gcp/install_upi.md

```diff
@@ -0,0 +1,594 @@
+# Install: User Provided Infrastructure (UPI)
+
+The steps for performing a UPI-based install are outlined here. Several
+[Deployment Manager][deploymentmanager] templates are provided to assist in
+completing these steps or to help model your own. You are also free to create
+the required resources through other methods; the templates are just an
+example.
+
+## Create ignition configs
```

Comment:
nit: "ignition" -> "Ignition", since it's a proper name.

Reply:
Done.